shell/browser/api/electron_api_protocol.cc

```diff
@@ -194,60 +193,59 @@ const char* const kBuiltinSchemes[] = {
     "about", "file", "http", "https", "data", "filesystem",
 };
 
+}  // namespace
+
+Protocol::Protocol(v8::Isolate* isolate, ProtocolRegistry* protocol_registry)
```

Comment:
Maybe also remove the unused isolate parameter

Reply:
713dc7cdb17214be62a00226b67af7328641b80e